eks/main.tf

```diff
@@ -6,7 +6,7 @@
 terraform {
   backend "s3" {
     bucket = "terramoz-eks-state"
-    key = "us-east-1/afrank-0/terraform.tfstate"
+    key    = "us-east-1/afrank-0/terraform.tfstate"
```

Comment:
I will say that this needs to change

Reply:
yeah I will offer a giant WTF to terraform for not letting you use variables in the terraform {} block.

Reply:
Its less than ideal but might have to do for now